rfcs/20211118-support-implicit-concat/README.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Support of implicit concat
+
+## Introduction
+
+This document explores a way of supporting implicit concat. Implicit concat is a
+concat primitive which is done without any additional copies just by storing
+correspondent parts into right spots of final tensor. Such approach allows to
+save at least one memory pass over the final output. Sometimes it may be two or
+more.
+
+### Motivation
+
+Since the compute powers are only increasing without supporting proper memory
+bandwidth, most optimizations target saving work with memory and move more
+in a compute field. Same desire here. We want to use intelligence to save on
+putting smaller memories into a bigger one just with right format.
+
+## Proposal
+
+Let's start with the fact that such optimization cannot be provided by oneDNN
+automatically - it must be enabled on user side manually due to several
```

Comment:
I think this is primarily because the user sets up the memory. In case of implicit concat, memory placement should be adjusted accordingly.

Reply:
I wanted to emphasize that API-wise this is not possible. I've updated this section and mentioned a case of memory management. Thanks.